lib/src/svg/xml_parsers.dart

```diff
@@ -346,16 +346,18 @@ DrawableStyle parseStyle(
     final Matrix4 transform = parseTransform(
       getAttribute(attributes, 'transform'),
     );
-    if (multiplyTransformByParent && parentStyle?.transform != null) {
-      if (transform == null) {
-        rawTransform = parentStyle.transform;
+    if (transform != null) {
```

Comment:
I saw this this morning and have a different fix - I believe this would cause problems if the transform is null and you want the parent transform.

Reply:
Yea. I was wondering about that. I originally changed the `else` to an `else if (transform != null)`, but then decided to move it up.

Happy to nuke this pr if you have something better already or I can change it back to the `else if`. :)

Reply:
My solution was to just change `transform.storage` to `transform?.storage`.